src/pages/Donate.jsx

```diff
@@ -1,101 +1,67 @@
-            const response = await GetAllClubs();
-            setClubData(response);
-        };
-        fetchClubData();
-    }, []);
+const Donate = () => {
+  document.title = "Milan | Donate the needy";
 
+  const [clubData, setClubData] = useState([]);
 
-    const loadScript = (src) => {
-        return new Promise((resolve) => {
-            const script = document.createElement("script");
-            script.src = src;
-            script.onload = () => {
-                resolve(true);
-            };
-            script.onerror = () => {
-                resolve(false);
-            };
-            document.body.appendChild(script);
-        });
+  useEffect(() => {
+    const fetchClubData = async () => {
+      const response = await GetAllClubs();
+      setClubData(response);
     };
-
-    useEffect(() => {
-        loadScript("https://checkout.razorpay.com/v1/checkout.js");
+    fetchClubData();
+  }, []);
+
+  //   Authentication state check !
+  const AuthState = () => {
+    const [login, setLogin] = useState(
+      Cookies.get("token") || Cookies.get("club")
+    );
+    return login;
+  };
+
+  const loadScript = (src) => {
+    return new Promise((resolve) => {
+      const script = document.createElement("script");
+      script.src = src;
+      script.onload = () => {
+        resolve(true);
+      };
+      script.onerror = () => {
+        resolve(false);
+      };
+      document.body.appendChild(script);
     });
+  };
 
+  useEffect(() => {
+    loadScript("https://checkout.razorpay.com/v1/checkout.js");
+  });
 
-    return (
-        <>
-
-            <Navbar />
-
-            <ToastContainer
-                position="top-right"
-                autoClose={1000}
-                hideProgressBar={false}
-                newestOnTop={false}
-                closeOnClick
-                rtl={false}
-                pauseOnFocusLoss
-                draggable
-                pauseOnHover
-                closeButton={false}
-            />
-
-            <div id="donate_banner"
-                className="container"
-            >
-
-                <div id="donateCol2">
-                    <img src={donate_image1} alt="" className="donate_img" />
-                </div>
-
-
-                <div
-                    id="donatecol_1"
-                    className="d-flex flex-column justify-content-center align-items-start me-5"
-                >
-                    <h1 className="mb-4">Yes, you help live !!</h1>
-                    <p>Donations does play an important part as our annual funds, donations from your ends helps thousands of unfortunate people live their lives. </p>{" "}
-
-                    <p>Choose any club, donate whatever you want, even 5 rupees helps !</p>
-
-
-                </div>
-
-            </div>
-
+  return (
+    <>
+      {AuthState() && <Navbar />}
 
-            <hr className="container" />
+      {AuthState() ? <DonateUs /> : <Banner />}
 
```

Comment:
This is where we are getting a error, this is bad for the UX

![image](https://user-images.githubusercontent.com/72851613/194480988-0d41f5ab-78b4-4c78-a3bb-2c06e6366a4f.png)


If someone without logging in are visiting the page this is what they see.
This might make the impression of the site being broken

Reply:
Okay, so if not logged in then they'll be redirected to the user login page with the mentioned toast. 

That's the requirement right ?

Reply:
Yes !